README.md

```diff
@@ -5,9 +5,9 @@ Canary is a Python3 bot designed for the McGill University Community Discord Ser
 
 ## Build Statuses
 
-| Master  | [![Build Status](https://github.com/Idoneam/Canary/workflows/YAPF%20Formatting%20Check/badge.svg?branch=master)](https://github.com/Idoneam/Canary/actions?query=branch%3Amaster) |
+| **Master**  | [![Poetry Production Dependency Installation](https://github.com/idoneam/Canary/workflows/Poetry%20Production%20Dependency%20Installation/badge.svg)](https://github.com/Idoneam/Canary/actions?query=branch%3Amaster+workflow%3Aformatting) |
```

Comment:
missing one `Idoneam` to `idoneam` change there

Reply:
Resolved in 8df0aa7